### PR TITLE
Photo & Caption components created

### PR DESCRIPTION
**Pull request by opmanual**
Photo & Caption components were created

### CODE REVIEW

File: src/post/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Post extends React.Component {
+
```
Comment (justinyum98) [inline src/post/index.js]: Delete this empty line here; it's not necessary.

File: src/post/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    render() {
+        const { post } = this.props;
+        const { photoUrl, caption } = post;
+
+        return (
+            <>
+                <Photo photoUrl={photoUrl} />
+                <Caption  caption={caption} />
```
Comment (justinyum98) [inline src/post/index.js]: Delete the extra space between `Caption` and `caption={caption}`

File: src/post/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+class Post extends React.Component {
+
+    render() {
+        const { post } = this.props;
+        const { photoUrl, caption } = post;
+
+        return (
+            <>
+                <Photo photoUrl={photoUrl} />
```
Comment (minybrandon) [inline src/post/index.js:12]: you forgot to import photo and caption component

File: src/post/caption.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+class Caption extends React.Component {
+    render() {
+        <p>
+            {this.props.caption}
+        </p>
+    }
+}
+
+export default Caption;
```
Comment (justinyum98) [inline src/post/caption.js:11]: Add a newline after the last line of code.

File: src/post/index.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+                <Caption  caption={caption} />
+            </>
+        )
+    }
+}
+
+export default Post;
```
Comment (justinyum98) [inline src/post/index.js:19]: Add a new line at the end of the file.

**Review by minybrandon — CHANGES_REQUESTED** [review]
fix plz

**Review by justinyum98 — CHANGES_REQUESTED** [review]
I suggested a couple revisions regarding code style, or changes that will make the code more readable.

File: src/post/caption.js (hunk 4 above)
Comment (sangyum) [reply src/post/caption.js:11]: 👍

File: src/post/index.js (hunk 3 above)
Comment (sangyum) [reply src/post/index.js:12]: 🤔

File: src/post/index.js (hunk 2 above)
Comment (sangyum) [reply src/post/index.js]: @justinyum98  is on fire! 🔥

File: src/post/photo.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+class Photo extends React.Component {
+    render() {
+        return (
+            <img src={this.props.photoUrl} />
+        )
+    }
+}
+
+export default Photo;
```
Comment (sangyum) [inline src/post/photo.js:11]: Add a new line here as well.

**Review by sangyum — CHANGES_REQUESTED** [review]
left some comments

File: src/post/index.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import React from 'react';
```
Comment (sangyum) [inline src/post/index.js]: remove this line

**Review by sangyum — CHANGES_REQUESTED** [review]
more comments

**Review by sangyum — APPROVED** [review]

**Review by minybrandon — APPROVED** [review]

**Review by minybrandon — APPROVED** [review]

**Review by justinyum98 — APPROVED** [review]
👍

**Review by minybrandon — APPROVED** [review]

**Pull request merged**